lessons/0_workflow/homework_workflow.md

```diff
@@ -0,0 +1,57 @@
+# Homework Workflow
+
+## Steps to submit your homework
+1. Check if you are in your master branch.
+```
+git branch
+```
+
+2. Keep your local repository updated.
+```
+git pull origin master && git fetch
+```
+
+3. Create a NEW branch for your homework from master.
+```
+git checkout -b homework_1
+```
+
+4. Commit your changes in your new branch.
```

Comment:
As I see from checkout we go directly to commit. Guess a step with 'do your changes' and 'git add' your changes shall be added here